etc/.golangci.yaml

```diff
@@ -17,43 +18,50 @@ linters:
     - contextcheck
     - cyclop
     - deadcode
+    - depguard
```

Comment:
This complained about a lot (all?) of our github imports. I suppose it expects one to create an allow-list of acceptable third party libraries.